drivers/dac/ad5791/ad5791.c

```diff
@@ -210,6 +211,26 @@ int32_t ad5791_get_register_value(struct ad5791_dev *dev,
 	return data_read;
 }
 
+/**
+ * @brief Get register value standard function.
+ * @param [in] dev - Pointer to the driver handler.
+ * @param [in] addr - Address of the register.
+ * @param [out] value - Pointer to the value of the register.
+ * @return 0 in case of success, error code otherwise.
+ */
+int32_t ad5791_get_register_value2(struct ad5791_dev *dev, uint8_t addr,
```

Comment:
In this case, I would only update the original ad5791_get_register_value() function instead of creating a wrapper.